cogs/modmail.py

```diff
@@ -1212,6 +1212,30 @@ async def logs_closed_by(self, ctx, *, user: User = None):
         session = EmbedPaginatorSession(ctx, *embeds)
         await session.run()
 
+    @logs.command(name="id")
+    @checks.has_permissions(PermissionLevel.SUPPORTER)
+    async def logs_id(self, ctx, key: str):
+        """
+        Get the log link for the specified log ID
+        """
+        icon_url = ctx.author.avatar.url
+
+        logs = await self.bot.api.find_log_entry(key)
+
+        if not any(not log["open"] for log in logs):
+            embed = discord.Embed(
+                color=self.bot.error_color,
+                description=f"Log entry `{key}` not found.",
+            )
+            return await ctx.send(embed=embed)
+
+        logs = reversed(log for log in logs if not log["open"])
```

Comment:
This line triggers a TypeError: `TypeError: 'generator' object is not reversible`

Reply:
I did not face that error when I tested it, I followed the `logs` command to maintain consistency
https://github.com/modmail-dev/modmail/blob/1f3bd0e9fcecb0be1feb217f174b47bb1d5e2251/cogs/modmail.py#L1184

Not sure if I am missing something

Reply:
It is throwing the error now for some reason yeah, lemme go have a look